### PR TITLE
specs-actors v0.6

### PR DESCRIPTION
**Pull request by magik6k**
Depends on:
- [x] https://github.com/filecoin-project/filecoin-ffi/pull/107

### CODE REVIEW

File: ffiwrapper/config.go [hunk 1]
```diff
@@ -7,31 +7,27 @@ import (
...
 
 type Config struct {
-	SealProofType abi.RegisteredProof
+	SealProofType abi.RegisteredSealProof
 
 	_ struct{} // guard against nameless init
 }
 
 func sizeFromConfig(cfg Config) (abi.SectorSize, error) {
-	if cfg.SealProofType == abi.RegisteredProof(0) {
```
Comment (laser) [inline ffiwrapper/config.go:16]: @magik6k - Why did you remove this if-statement? Is it no longer needed now that we have the new types?

**Comment by laser** [comment]
@magik6k - Note: The `mod-tidy-check` is needs to be run.

File: ffiwrapper/config.go (hunk 1 above)
Comment (magik6k) [reply ffiwrapper/config.go:16]: The 2K seal proof type is `0`

**Comment by anorth** [comment]
👍 I just checked in here to make sure I'm keeping track of the integration difficulty of actors changes for downstream consumers. Please feel free to tag me on "update actors" PRs and lmk if things are unnecessarily inconvenient. We do, of course, need to actually change some things now and then.
